storybook/stories/support/StoryLayout.js

```diff
@@ -153,31 +158,42 @@ export default class StoryLayout extends Component<Props> {
     activeSidebarCategory: string,
     activeWallet: Wallet,
     activeNavItem: string
-  ) => (
-    <TopBar
-      onToggleSidebar={() => {
-        runInAction(() => {
-          this.isShowingSubMenus = !this.isShowingSubMenus;
-        });
-      }}
-      formattedWalletAmount={formattedWalletAmount}
-      currentRoute={`/wallets/${activeWallet.id}/${activeNavItem}`}
-      activeWallet={
-        activeSidebarCategory === '/wallets' && activeNavItem !== 'empty'
-          ? activeWallet
-          : null
-      }
-      showSubMenuToggle
-      showSubMenus={this.isShowingSubMenus}
-    >
-      <NodeSyncStatusIcon
-        networkStatus={{
-          isSynced: true,
-          syncPercentage: 100,
+  ) => {
+    const notification = (
+      <LegacyNotification onLearnMore={() => null} onMove={() => null} />
+    );
+    const topbar = (
+      <TopBar
+        onToggleSidebar={() => {
+          runInAction(() => {
+            this.isShowingSubMenus = !this.isShowingSubMenus;
+          });
         }}
-        isProduction
-        isMainnet
-      />
-    </TopBar>
-  );
+        formattedWalletAmount={formattedWalletAmount}
+        currentRoute={`/wallets/${activeWallet.id}/${activeNavItem}`}
+        activeWallet={
+          activeSidebarCategory === '/wallets' && activeNavItem !== 'empty'
+            ? activeWallet
+            : null
+        }
+        showSubMenuToggle
+        showSubMenus={this.isShowingSubMenus}
+      >
+        <NodeSyncStatusIcon
+          networkStatus={{
+            isSynced: true,
+            syncPercentage: 100,
+          }}
+          isProduction
+          isMainnet
+        />
+      </TopBar>
+    );
+
+    return activeWallet.isLegacy ? (
```

Comment:
@yakovkaravelov I don't like this approach. Can't we make the legacy notification to be a part of the TopBar component? That should be much cleaner...

Reply:
Fixed this.